### PR TITLE
ansible-test: update http-test-container to 3.2.0

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
https://github.com/ansible/http-test-container/releases/tag/3.1.0

Ref: https://github.com/ansible-collections/community.crypto/pull/768#issuecomment-2166653041

##### ISSUE TYPE
- Feature Pull Request
- Test Pull Request

### CODE REVIEW

File: test/lib/ansible_test/_internal/commands/integration/cloud/httptester.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ class HttptesterProvider(CloudProvider):
     def __init__(self, args: IntegrationConfig) -> None:
         super().__init__(args)
 
-        self.image = os.environ.get('ANSIBLE_HTTP_TEST_CONTAINER', 'quay.io/ansible/http-test-container:3.0.0')
+        self.image = os.environ.get('ANSIBLE_HTTP_TEST_CONTAINER', 'quay.io/ansible/http-test-container:3.1.0')
```
Comment (mattclay) [inline test/lib/ansible_test/_internal/commands/integration/cloud/httptester.py]:
```suggestion
        self.image = os.environ.get('ANSIBLE_HTTP_TEST_CONTAINER', 'quay.io/ansible/http-test-container:3.2.0')
```

File: changelogs/fragments/83469-http-test-container.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "ansible-test - update HTTP test container to 3.1.0 (https://github.com/ansible/ansible/pull/83469)."
```
Comment (mattclay) [inline changelogs/fragments/83469-http-test-container.yml]:
```suggestion
  - "ansible-test - update HTTP test container to 3.2.0 (https://github.com/ansible/ansible/pull/83469)."
```

**Review by mattclay — CHANGES_REQUESTED** [review]
I've built a 3.2.0 container, we should use that instead of 3.1.0.

**Review by mattclay — APPROVED** [review]

**Comment by felixfontein** [comment]
@mattclay thanks for reviewing and merging!

**Pull request merged**